tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot

```diff
@@ -390,6 +390,156 @@ Container network - space in network name valid
     Cleanup VCH Bridge Network  %{VCH-NAME}
     Cleanup VIC Appliance On Test Server
 
+Container Firewalls
+    Set Test Environment Variables
+    # Attempt to cleanup old/canceled tests
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+
+    ${out}=  Run  govc host.portgroup.remove bridge
+    ${out}=  Run  govc host.portgroup.remove open-net
+    ${out}=  Run  govc host.portgroup.remove closed-net
+    ${out}=  Run  govc host.portgroup.remove published-net
+    ${out}=  Run  govc host.portgroup.remove outbound-net
+    ${out}=  Run  govc host.portgroup.remove peers-net-1
+    ${out}=  Run  govc host.portgroup.remove peers-net-2
+
+    Log To Console  Create port groups
+    ${out}=  Run  govc host.portgroup.add -vswitch vSwitchLAN bridge
+    ${out}=  Run  govc host.portgroup.add -vswitch vSwitchLAN open-net
+    ${out}=  Run  govc host.portgroup.add -vswitch vSwitchLAN closed-net
+    ${out}=  Run  govc host.portgroup.add -vswitch vSwitchLAN published-net
+    ${out}=  Run  govc host.portgroup.add -vswitch vSwitchLAN outbound-net
+    ${out}=  Run  govc host.portgroup.add -vswitch vSwitchLAN peers-net-1
+    ${out}=  Run  govc host.portgroup.add -vswitch vSwitchLAN peers-net-2
+
+    ${createcommand}=  catenate  SEPARATOR=\ \ 
+    ...  bin/vic-machine-linux create --debug 1 --name=%{VCH-NAME}
+    ...  --target=%{TEST_URL}%{TEST_DATACENTER} --thumbprint=%{TEST_THUMBPRINT}
+    ...  --user=%{TEST_USERNAME} --image-store=%{TEST_DATASTORE} --password=%{TEST_PASSWORD}
+    ...  --force=true --bridge-network=bridge --compute-resource=%{TEST_RESOURCE} --no-tlsverify
+    ...  --insecure-registry harbor.ci.drone.local
+    ...  --container-network open-net --container-network-firewall open-net:open
+    ...  --container-network closed-net --container-network-firewall closed-net:closed
+    ...  --container-network outbound-net --container-network-firewall outbound-net:outbound
+    ...  --container-network published-net --container-network-firewall published-net:published
+    ...  --container-network peers-net-1 --container-network-firewall peers-net-1:peers
+    ...  --container-network-ip-range peers-net-1:10.10.10.0/24 --container-network-gateway peers-net-1:10.10.10.1/24
+    ...  --container-network peers-net-2 --container-network-firewall peers-net-2:peers
+    ...  --container-network-ip-range peers-net-2:192.168.0.0/16 --container-network-gateway peers-net-2:192.168.0.1/16
```

Comment:
Yes you should likely not show that since the user did not explicitly specify that. You should have a test case for this though; it should behave _as if_ published was specified.